eth/downloader/queue.go

```diff
@@ -49,35 +56,73 @@ type fetchRequest struct {
 	Time    time.Time           // Time when the request was made
 }
 
+// fetchResult is the assembly collecting partial results from potentially more
```

Comment:
This sentence is a bit funky, I guess it should be either "assembly of partial results" or "is collecting partial results"